tests/suite/system_suite_test.go

```diff
@@ -173,6 +175,10 @@ func setup(cfg setupConfig, extraInstallArgs ...string) {
 		}
 		installCfg.ImageTag = *imageTag
 		installCfg.ImagePullPolicy = *imagePullPolicy
+	} else {
+		if version == "edge" {
+			installCfg.ChartVersion = "0.0.0-edge"
+		}
```

Comment:
Since the default behavior is to run the tests using the local chart, does setting this version affect that in any way?

Reply:
when there's the `oci://` path and `edge` we set this, otherwise the `--version` is not used

Reply:
Ah right, I was reading the conditions wrong.